src/main/java/org/openpnp/machine/reference/feeder/wizards/ReferenceTubeFeederConfigurationWizard.java

```diff
@@ -19,13 +19,13 @@
 
 package org.openpnp.machine.reference.feeder.wizards;
 
-import org.openpnp.machine.reference.feeder.ReferenceTubeFeeder;
+import org.openpnp.machine.reference.ReferenceFeeder;
 
 public class ReferenceTubeFeederConfigurationWizard
         extends AbstractReferenceFeederConfigurationWizard {
-    private final ReferenceTubeFeeder feeder;
+    private final ReferenceFeeder feeder;
```

Comment:
For adaptive, you can either remove it, create a new stage, or add a boolean that turns it on. Using a special number makes it hard to understand for users and the purpose of the pipeline stage system is to be self documenting and easy to understand. Easiest would just be to add a boolean property to turn it on and off. boolean adaptive = false. 

For the wizard, yes, the Abstract one is an abstract class, so it cannot be instantiated directly. You have to extend it but no additional code is required - just extend and provide a constructor. If you want a generic wizard you can extend it and call it ReferenceFeederConfigurationWizard